### PR TITLE
Api proposal

### PR DESCRIPTION
**Pull request by maasencioh**
@lpatiny @targos @stropitek what do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,58 @@
...
+
+```tsx
+const Example = () => (
+  <Plot
+    width={500}
+    height={500}
+    colorScheme={d3.schemePaired}
+    margin={{ right: 200 }}
+  >
+    <PlotHeading
```
Comment (lpatiny) [inline README.md:18]: I would rename to Heading and remove the property 'top'. If it is a heading it should be on the top.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,58 @@
...
+      title="Electrical characterization"
+      subtitle="current vs voltage"
+      titleStyle={}
+      substitleStyle={}
+      titleClass=""
+      substitleClass=""
+      position="top"
+    />
+    <PlotLineSeries
```
Comment (lpatiny) [inline README.md:27]: Could be renamed to LineSeries

File: README.md [hunk 3]
```diff
@@ -1,21 +1,58 @@
...
+      label="Vg = 7V"
+    />
+    <PlotLineSeries
+      data={{ x: [0, 1, 2, 3, 4, 5], y: [0, 2, 4, 6, 6, 6] }}
+      lineStyle={{ stroke: 'blue', strokeWidth: 2 }}
+      markerStyle={}
+      displayMarker={true}
+      label="Vg = 3V"
+    />
+    <PlotXAxis label="Drain voltage [V]" showTicks={true} tickStep={1} />
```
Comment (lpatiny) [inline README.md:40]: could also be renamed to XAxis, YAxis, Legend

File: README.md (hunk 1 above)
Comment (maasencioh) [reply README.md:18]: If I have several plots I could want to have this on the bottom, and I like the `ComponentSubcomponent` style for subcomponents, in order to make it easy to understand when you imported (Header could be your layout header, menu header, etc)

**Comment by maasencioh** [comment]
Another option will be to use `Plot.Heading` or `Plot.XAxis` for example, but I prefer clear names

**Review by targos — APPROVED** [review]

**Pull request merged**